integrated_channels/degreed/client.py

```diff
@@ -40,6 +40,11 @@ def __init__(self, enterprise_configuration):
         self.session = None
         self.expires_at = None
 
+    def create_assessment_reporting(self, user_id, payload):
+        """
+        Not implemented yet.
```

Comment:
pylint was complaining because the base level client has the non-implemented method for `create_assessment_reporting`, following the pattern for all the `client` methods. So I believe all the other integrated channels need this as of right now?

Reply:
ahhh I didn't realize that was also missing in the base class. My b.